src/libxc/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+set(CMAKE_C_FLAGS "-c -I. -O0 -g -fPIC") # ${CCC}
```

Comment:
It's not great that flags like this are being specified in a bunch of different places.  What if you use a different compiler such as Intel that needs different optimization flags?  It would be a lot better to make constants that are defined to the opt and non-opt flags (for each language) and then activate them when needed.  This is how I've done it in Amber (except on a source file or target basis, not at directory scope).

Reply:
This line seems to be unnecessary...